web/admin/application/configs/klear/BillableCallsClientList.yaml

```diff
@@ -89,6 +94,20 @@ production:
             price: 4
             ratingPlanName: 6
             destinationName: 6
+  dialogs:
+    <<: *billableCalls_dialogsLink
+
+  commands: &exportBillableCallsToCsv_commandsLink
+    exportBillableCallsToCsv_command: &exportBillableCallsToCsv_commandLink
+      <<: *BillableCalls
+      title: _("Export to CSV")
+      label: true
+      module: default
+      controller: klear-custom-billable-calls
+      action: export-to-csv
+      class:  ui-silk-page-white-database
+      forcedValues:
+        "company": ${auth.companyId}
```

Comment:
this is equal to _<<: *forcedCompany_

Reply:
Not exactly: forcedCompany sets `"self::company": ${auth.companyId}` and `self` is not automatically resolved in commands